### DISCUSSION TITLE
Window Split to next Tab

### DISCUSSION THREAD

**Discussion by Randalix** [kovidgoyal/kitty#6452] [Q&A]
How can I move a window split to a new/existing tab?
I hope I did not miss it in the docs...

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/overview/#detach-window

On Wed, Jul 12, 2023 at 12:34:21AM -0700, Randalix wrote:
> How can I move a window split to a new/existing tab?
> I hope I did not miss it in the docs...
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6452
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
